src/main/java/teammates/ui/controller/XlsFileDownloadResult.java

```diff
@@ -44,7 +44,8 @@ private XSSFWorkbook getWorkBook() {
         int rowNumber = 0;
```

Comment:
As much as possible, try to write standalone methods. i.e. all input come as parameters and the only output is the return value. Doing so improves testability. For example, you can have a utility method `convertToWorkbook` that takes a string as the input and returns the corresponding workbook. That method should be unit tested to ensure it works for all cases. empty content, comma inside cell, etc.